dumper/app/build.gradle

```diff
@@ -25,6 +29,9 @@ plugins {
 }
 
 apply plugin: 'com.github.jk1.dependency-license-report'
+apply plugin: 'nebula.info'
+version = '1.0.8'
```

Comment:
I think we don't want version or group here - version is specified at release-build time, and group is specified ... where? cc @paolomorandini for explicit review please